WMFData/Sources/WMFData/Data Controllers/Article Tabs/WMFArticleTabsDataController.swift

```diff
@@ -417,7 +417,9 @@ public protocol WMFArticleTabsDataControlling {
             try self.deleteAllTabs(moc: moc)
         }
         
-        _ = try? await self.createArticleTab(initialArticle: nil, setAsCurrent: true)
+        if !moreDynamicTabsGroupBEnabled && !moreDynamicTabsGroupCEnabled {
```

Comment:
You can simplify this to `!needMoreDynamicTabs`, same cooment from another PR in case you do that one first